nuxt.config.js

```diff
@@ -85,7 +85,29 @@ export default {
   /*
    ** Nuxt.js modules
    */
-  modules: ['@nuxt/content', '@nuxtjs/axios', '@nuxtjs/i18n'],
+  modules: [
+    '@nuxt/content',
+    '@nuxtjs/axios',
+    '@nuxtjs/i18n',
+    '@nuxtjs/sitemap',
+    '@nuxtjs/robots',
+    [
+      'nuxt-social-meta',
+      {
+        url: 'https://new.an-nadaa.com/',
+        title: 'An-nadaa project, build your hereafter today',
+        site_name: 'An-nadaa Educational foundation',
+        description:
+          'An-nadaa project, build for your hereafter today,You can change the lives of millions by supporting educational and welfare projects of An-Nadaa that are bringing a huge positive impact to the community.',
```

Comment:
Can we change all the descriptions to this:
```
Build for your hereafter today with An-Nadaa. You can change the lives of millions by supporting educational and welfare projects of An-Nadaa that are bringing a huge positive impact to the community.
```

Reply:
For each page, we need to have a clear description for it, I was adding custom description message for faq, blog... just to make the user on the page scope when he search for an nadaa blog for example, instead of default message, I will change the nuxt config description then remove the description for the home page